libs/auth/react/src/lib/AuthSettings.ts

```diff
@@ -38,6 +38,13 @@ export interface AuthSettings extends UserManagerSettings {
   userStorePrefix?: string
 }
 
+export interface AuthSettingsWithScopes extends AuthSettings {
+  /**
+   * Allow to pass an array of scope needed for the client.
+   */
+  scopes: string[]
```

Comment:
`scopes` feels kind of strange to me. i know it's a list and all that, but you're just so used to always using `scope` - scope is a collection of permissions, it's not a collection of scopes. right?

Reply:
Yes you are right, I will change that 👍